README.md

```diff
@@ -64,6 +64,50 @@ services:
 
 then start with `docker-compose up -d`
 
+**Installation with systemd** (on ubuntu 18.04)
+
+Prerequisites:
+
+```bash
+Add a repository with the latest Version of golang and install the prerequisites:
+sudo add-apt-repository ppa:longsleep/golang-backports
+sudo apt update && sudo apt upgrade
+sudo apt install build-essential git sqlite libtag1-dev ffmpeg libasound-dev golang
+```
+
+Download and compile gonic in the home directory  
+
+```bash
+go get go.senan.xyz/gonic/cmd/gonic
+```
+
+Add a gonic user and create a directory for the server:
+
+```bash
+sudo mkdir -p /var/gonic/
+sudo adduser --disabled-login --gecos "" gonic
+sudo mv go/* /var/gonic/
```

Comment:
sorry, another question. what does this `mv` do?

Reply:
go get install the files by deafult in the folder go in your homedir, this mv changed the path from ~/go/ to /var/gonic.

Reply:
ahh i see. well just incase the user has other go software installed, it might be better to change to something like `mv go/bin/gonic /var/gonic`? then change the `ExecStart` below to remove the `bin/`? 
what do you think? this way we only copy the binary